src/Forms/ConfirmedPasswordField.php

```diff
@@ -708,6 +780,18 @@ public function getRequireStrongPassword(): bool
         return $this->requireStrongPassword;
     }
 
+    /**
+     * Set whether the field is on a Member form
+     * If this is set to true then setRequireStrongPassword() should be set to false
+     * Thisi is because we do not want to use this fields validate() method to validate
+     * the password, instead we want to validate using MemberAuthenticator
```

Comment:
```suggestion
    /**
     * Set whether the field is on a Member form. This is used to determine whether and how to evaluate password strength for this form field.
```
`setRequireStrongPassword()` true while we're on a member form is weird, but perfectly valid.

Reply:
Updated